CoreFoundation/String.subproj/CFCharacterSet.c

```diff
@@ -1547,7 +1558,7 @@ CFCharacterSetRef CFCharacterSetCreateInvertedSet(CFAllocatorRef alloc, CFCharac
 Boolean _CFCharacterSetInitMutable(CFMutableCharacterSetRef cset) {
     if (!__CFCSetGenericInit(cset, __kCFCharSetClassBitmap| __kCFCharSetIsMutable)) return false;
     __CFCSetPutBitmapBits(cset, NULL);
-    __CFCSetPutHasHashValue(cset, true);
+    __CFCSetPutHasHashValue(cset, true); // Hash value is 0
```

Comment:
Was this comment copied and pasted from above? I thought the comment was regarding the condition rather than the action of the line, so I'm not sure it makes sense to add it here - but @phausler would know more, I suspect.

Reply:
I simply added it for the consistency, it all the similar places the same comment exist. I actually tend to have "no comment" policy, so I would also go for removing it.